### PR TITLE
Feature/progress style

### PR DESCRIPTION
**Pull request by jfrankl**

### CODE REVIEW

**Comment by jfrankl** [comment]
On mobile, modal functions like normal:

image

On desktop, it expands to a sidebar or panel that is always open:

image

File: src/nyc_trees/sass/partials/_globals.scss [hunk 1]
```diff
@@ -299,3 +298,40 @@ label {
 .map-small {
   padding-top: 40%; // Makes height 40% of width
 }
+
+.map {
```
Comment (maurizi) [inline src/nyc_trees/sass/partials/_globals.scss]: I introduced a `partials/_map.scss` file recently, with a `map` class.  We should either move this class there or removing the `_map.scss` file.

File: src/nyc_trees/sass/partials/_globals.scss [hunk 2]
```diff
@@ -299,3 +298,40 @@ label {
...
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+@media (min-width: 768px) {
```
Comment (maurizi) [inline src/nyc_trees/sass/partials/_globals.scss]: Should this use a `$screen-` variable?

File: src/nyc_trees/sass/partials/_globals.scss (hunk 1 above)
Comment (jfrankl) [inline src/nyc_trees/sass/partials/_globals.scss]: I added this to _map.scss.

**Comment by maurizi** [comment]
When I tested this on chrome, the legend prevents clicking on the map in the area below it.

werid_slice

**Comment by jfrankl** [comment]
@maurizi, I fixed that glitch. Let me know if you have anything else.

**Comment by maurizi** [comment]
+1

**Comment by maurizi** [comment]
I don't think it has to be fixed in this PR, but as an FYI this does mess up the styling a bit on the reservations map, which currently shares `legend.html` (though that page needs styling work right now anyways)

We'll need to find a way to change the text on the reservations page (maybe inherit from a shared template and override a block?)
screenshot from 2015-01-25 14 41 30

**Pull request merged**